src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -274,7 +289,7 @@ def load_arguments(self, _):
         c.argument('disable_scope_strict_match', options_list=['--disable-scope-strict-match', '-i'], action='store_true', help='Include policy exemptions either inherited from parent scope or at child scope.')
         c.argument('display_name', help='Display name of the policy exemption.')
         c.argument('description', help='Description of policy exemption.')
-        c.argument('exemption_category', options_list=['--exemption-category', '-e'], help='The policy exemption category of the policy exemption', arg_type=get_enum_type(ExemptionCategory))
+        c.argument('exemption_category', options_list=['--exemption-category', '-e'], help='The policy exemption category of the policy exemption', arg_type=get_enum_type(['ExemptionCategory']))
```

Comment:
Please define valid values in the enumeration instead of the enumeration name
```suggestion
        c.argument('exemption_category', options_list=['--exemption-category', '-e'], help='The policy exemption category of the policy exemption', arg_type=get_enum_type(['Waiver', 'Mitigated']))
```

Reply:
> <html>
<body>
<!--StartFragment-->

Type | Test Case | Error Message | Line
-- | -- | -- | --
Failed | test_resource_policyexemption_default | E SystemExit: 2 | azure/cli/command_modules/resource/tests/latest/test_resource.py:4358
Failed | test_resource_policyexemption_management_group | E SystemExit: 2 | azure/cli/command_modules/resource/tests/latest/test_resource.py:4363

<!--EndFragment-->
</body>
</html>

Actually, the current CI issue is introduced by this error usage above